test/test_cpp_extensions_mtia_backend.py

```diff
@@ -7,24 +7,16 @@
 import torch
 import torch.testing._internal.common_utils as common
 import torch.utils.cpp_extension
-from torch.testing._internal.common_utils import (
-    IS_ARM64,
-    IS_LINUX,
-    skipIfTorchDynamo,
-    TEST_CUDA,
-    TEST_PRIVATEUSE1,
-    TEST_XPU,
-)
-from torch.utils.cpp_extension import CUDA_HOME, ROCM_HOME
+from torch.testing._internal.common_utils import IS_ARM64, IS_LINUX, skipIfTorchDynamo
 
 
-# define TEST_ROCM before changing TEST_CUDA
-TEST_ROCM = TEST_CUDA and torch.version.hip is not None and ROCM_HOME is not None
-TEST_CUDA = TEST_CUDA and CUDA_HOME is not None
+# This TestCase should be mutually exclusive with other backends.
+acc = torch.accelerator.current_accelerator()
+is_skip = not (acc is None or acc.type == "mtia")
```

Comment:
@albanD I skipped this TestCase because it failed during periodic testing with a CUDA-built binary on a system without a GPU. Now it will be test only when `acc is None or acc is mtia`

Reply:
> CUDA-built binary on a system without a GPU

This is a valid use case! Many of our users grab cuda build from pypi (as it's the only build we have on pypi) even when they don't have an nvidia gpu.
It is a case we need to keep working properly !

Reply:
I see. What I meant is that MTIA and CUDA should be mutually exclusive, correct? So a binary built with CUDA shouldn't be running this TestCase.